manifests/init.pp

```diff
@@ -438,6 +480,123 @@
       enable  => $enable_pcsd_service,
       require => Package['pcs'],
     }
+
+    # Determine if this node should perform authorizations
+    case $manage_pcsd_auth_node {
+      'first': { $auth_node = $quorum_members[0] }
+      'last': { $auth_node = $quorum_members[length($quorum_members)-1] }
+      default: {}
```

Comment:
As $manage_pcsd_auth_node is set via a parameter with an Enum data type explicitly restricting it's possible values to 'first' and 'last' it shouldn't be possible for the default case to even hit. I only included it to avoid lint errors.

Is it worth adding a failure statement in a scenario like this?

Reply:
Well, IMHO no then, I'm not sure about the Puppet Style Guide.

Reply:
I can definitely add a comment for clarity but it looks like the default block should be there based on the style guide:
https://puppet.com/docs/puppet/5.5/style_guide.html#defaults-for-case-statements-and-selectors